pages/about-us/press/canary-statement/pt.md

```diff
@@ -0,0 +1,43 @@
+@title = 'Riseup usará emails criptografados em resposta a solicitações legais'
+@nav_title = 'Declaração sobre a Canária'
+@toc = false
+
+_16 de fevereiro de 2017_
+
+Após esgotar nossas opções legais, o Riseup decidiu recentemente cumprir dois mandados do FBI que estavam sob segredo de justiça, ao invés de desobedecê-los (o que teria resultado na prisão dos passarinhos do Riseup por um tempo e/ou no término da organização). O primeiro mandado tem relação com o endereço público de contato de um anel de extorsão DDoS internacional. O segundo tem relação com uma conta que usava um software de sequestro de dados (ransomware) para extorquir pessoas.
+
+Atividades de extorsão claramente violam ambos a carta de princípios e o espírito do contrato social 1 que temos com nossos usuários: Nós seguramos a sua onda se você não estiver realizando ações de exploração, misóginas, racistas e intolerantes.
```

Comment:
that's how the format is in that language, i mentioned things should be rechecked since some translations are different from the contents in english. this is a minor one if i recall correctly, i didn't tried to fix most of them as it was impossible for me to track them